src/styles/blocks/card.scss

```
@@ -0,0 +1,109 @@
+.container {
+  @include on1024;
+  padding: 50px 128px 72px;
+  display: grid;
+  grid-template-columns: 200px 200px 200px 200px;
+  grid-template-rows: 1fr 1fr;
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-property: transform;
+  transition-duration: 300ms;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__name {
+    margin: 0;
+    @include contentPaddingInline;
+    color: $main--color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__code {
+    @include contentPaddingInline;
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary--color;
+  }
+
+  &__reviews {
+    @include contentPaddingInline;
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+    &-comments {
+      align-self: center;
+      margin: 0;
+      padding: 0;
+      font-size: 10px;
+      line-height: 14px;
+      color: $main--color;
+    }
+  }
+
+  &__price {
+    @include contentPaddingInline;
+    margin-top: 24px;
+    margin-bottom: 0;
+    display: flex;
+    justify-content: space-between;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
```

if you make a variable for colors - make for all
```suggestion
    color: $c-color;
```